distrho/src/DistrhoPluginAU.cpp

```diff
@@ -87,31 +84,26 @@ class PluginAU : public AUEffectBase
     {
         DISTRHO_SAFE_ASSERT_RETURN(inScope == kAudioUnitScope_Global, kAudioUnitErr_InvalidParameter);
 
+        outParameterInfo.flags = kAudioUnitParameterFlag_IsReadable | kAudioUnitParameterFlag_IsWritable;
+
         // Name
         {
             const String& name = fPlugin.getParameterName(inParameterID);
             CFStringRef cfname = CFStringCreateWithCString(kCFAllocatorDefault, name.buffer(), kCFStringEncodingUTF8);
 
-            AUBase::FillInParameterName(outParameterInfo, cfname, true); // FIXME true or false??
+            AUBase::FillInParameterName(outParameterInfo, cfname, false);
```

Comment:
did you confirm that the memory from `CFStringCreateWithCString` does not need to be released?
(the 3rd argument is about freeing memory from the string)

Reply:
I was playing with this argument to see if params would work.  I didn't end up fixing it so this change seemed to be a no-op to me.

Reply:
but leaving it as false might lead to memory leaks.
I think only if something crashes we need to set it as false